### PR TITLE
[xml] Update german.xml v8.7

### PR DESCRIPTION
**Pull request by schnurlos**
Fix some shortcuts

### CODE REVIEW

**Comment by MarkusBodensee** [comment]
@schnurlos 
Since you are on it: In Line 423 and 424, item "11008" and "11009", you can remove the word "Nach" from the Item names. (The other items from this submenu don't have it either.) Because "nach" is already given in the Menu Entry one above which is in Line 92, item "window-sortby". See:
```
<Item id="11008" name="Nach Inhaltslänge aufsteigend"/>
<Item id="11009" name="Nach Inhaltslänge absteigend"/>
```

`<Item subMenuId="window-sortby" name="Sortieren nach"/>`

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/7221647b7490a223a0026a0184b4fa7990f281aa/PowerEditor/installer/nativeLang/german.xml#L424

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/7221647b7490a223a0026a0184b4fa7990f281aa/PowerEditor/installer/nativeLang/german.xml#L92

**Comment by donho** [comment]
@schnurlos 
Could you solve the conflict?

File: PowerEditor/installer/nativeLang/german.xml [hunk 1]
```diff
@@ -639,14 +639,15 @@ Translation note:
...
-					<Item id="2232" name="Unterstrichen als Standard setzen"/>
+					<Item id="2226" name="Standardfarbe Vordergrund für alle Stile erzwingen"/>
+					<Item id="2227" name="Standardfarbe Hintergrund für alle Stile erzwingen"/>
+					<Item id="2228" name="Schriftart als Standard für alle Stile erzwingen"/>
+					<Item id="2229" name="Schriftgröße als Standard für alle Stile erzwingen"/>
+					<Item id="2230" name="Fettschrift als Standard für alle Stile erzwingen"/>
+					<Item id="2231" name="Kursivschrift als Standard für alle Stile erzwingen"/>
+					<Item id="2232" name="Unterstrichen als Standard für alle Stile erzwingen"/>
 					<Item id="2234" name="Zu den Einstellungen"/>
+					<Item id="2235" name="Was bedeutet 'Globale Überschreibung'?"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/german.xml:650]: "Global override" should not be translated.

**Review by donho — CHANGES_REQUESTED** [review]
Restore "Global override"

**Review by donho — APPROVED** [review]

**Pull request closed without merging**